[cli] test command always shows help input

When running the `cft test run` command, if the tests fail the (verbose) help text is always printed. This is unhelpful and suppresses useful information.

The help text should only be printed if the incorrect number of arguments is supplied (or should never be printed at all).

```
$ cft test run TestSimpleExample --stage=init
INFO[01-21|15:07:21] using test-dir: . 
ok      github.com/terraform-google-modules/terraform-google-event-function/test/integration    0.230s [no tests to run]
# github.com/terraform-google-modules/terraform-google-event-function/test/integration/simple_example [github.com/terraform-google-modules/terraform-google-event-function/test/integration/simple_example.test]
simple_example/simple_example_test.go:35:3: protected_project_id declared but not used
simple_example/simple_example_test.go:36:3: dataset_name declared but not used
simple_example/simple_example_test.go:37:3: table_id declared but not used
simple_example/simple_example_test.go:38:3: public_project_id declared but not used
simple_example/simple_example_test.go:39:3: policy_id declared but not used
simple_example/simple_example_test.go:40:3: access_level_name declared but not used
simple_example/simple_example_test.go:41:3: regions_to_allow declared but not used
FAIL    github.com/terraform-google-modules/terraform-google-event-function/test/integration/simple_example [build failed]
FAIL
Error: error running command: exit status 2
Usage:
  cft test run [flags]
  

Flags:
  -h, --help           help for run
      --stage string   Test stage to execute (default is running all stages in order - init, apply, verify, teardown)

Global Flags:
      --test-dir string   Path to directory containing integration tests (default is computed by scanning current working directory)
      --verbose           Log output to stdout
      ```

--- a/cli/bptest/cmd.go
+++ b/cli/bptest/cmd.go
@@ -1,6 +1,8 @@
 package bptest
 
 import (
+	"os"
+
 	"github.com/jedib0t/go-pretty/v6/table"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
@@ -79,6 +81,12 @@ var runCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		return streamExec(testCmd)
+		// if err during exec, exit instead of returning an error
+		// this prevents printing usage as the args were validated above
+		if err := streamExec(testCmd); err != nil {
+			Log.Error(err.Error())
+			os.Exit(1)
+		}
+		return nil
 	},
 }

